site/src/components/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
This file should be under `sections` and not `components`
Create a folder named `Navbar` under `sections` and then create an index.js file and a `navbar.style.js` under it

Reply:
done :+1:

Reply:
done :+1: